src/Selenium2Library/keywords/_element.py

```diff
@@ -267,15 +267,31 @@ def element_text_should_be(self, locator, expected, message=''):
                           "in fact it was '%s'." % (locator, expected, actual)
             raise AssertionError(message)
 
-    def get_element_attribute(self, attribute_locator):
-        """Return value of element attribute.
+    def get_element_attribute(self, locator, attribute_name=None):
+        """Returns value of the element attribute.
 
-        `attribute_locator` consists of element locator followed by an @ sign
-        and attribute name, for example "element_id@class".
+        There are two cases how to use this keyword.
+
+        First, if only `locator` is provided, `locator` should consists of
+        element locator followed by an @ sign and attribute name.
+        This behavior is left for backward compatibility.
+
+        Example:
+        | ${id}= | Get Element Attribute | link=Link with id@id
+
+        Second, if `locator` and `attribute_name` are provided both, `locator`
```

Comment:
Lines look little bit too long, but I am doing review over mobile and therefore I am not totally sure about the line length. It's good to follow pep8[1] with the max line length. Could you check and fix it if there are errors.

There are two exceptions to the max line length. Keyword name and lines containing examples, like line 280, may exceed the line max length. This is because readability overrules in that case.

[1] https://www.python.org/dev/peps/pep-0008/#maximum-line-length